src/scss/components/_video.scss

```diff
@@ -0,0 +1,63 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0px;
+}
+
+.container__video {
+    padding: 80px 80px 88px 83px;
+}
+
+.info-video__section {
+    display: grid;
+    grid-template-columns: repeat(2, 1fr);
+    grid-template-rows: auto;
+}
+
+.info-video__paragraph {
+    grid-column: 1/2;
+    grid-row: 1/2;
+    align-self: center;
+}
+
+.paragraph__title--text{
+    @include fontStyle_video(var(--waikawa-gray), 24px, bold);
+    margin-bottom: 20px;
+}
+
+.paragraph__text--paragraph {
+    @include fontStyle_video(var(--cool-gray), 24px, normal);
+    max-width: 518px;
+}
+
+.video-section {
+    align-self: center;
+    margin-left: 25px;
+}
+
+.video-section__content {
+    width: 100%;
+    border-radius: 8px;
+}
+
+@media (min-width: 576px) and (max-width: 768px){
```

Comment:
Use the breakpoint mixin instead of this media query: `@include breakpoint('<viewport>')`. <viewport> can be 'sm', 'md', 'lg', 'xl'. (see the `mixin.scss` file for reference). Take in account that you would need to change your approach to be mobile first.

Reply:
I have added the mixins but the project have conflict whit the file Sass, this file take  the changes only the breakpoint when included mixin, however don´t take the main view styles. Help please, whit the media queries normal, I was able to solve this problem.